doc/classes/Object.xml

```diff
@@ -38,77 +42,119 @@
+				[b]Note:[/b] [method _property_can_revert] must also be overridden for this method to be called.
 			</description>
 		</method>
 		<method name="_set" qualifiers="virtual">
 			<return type="bool" />
 			<param index="0" name="property" type="StringName" />
 			<param index="1" name="value" type="Variant" />
 			<description>
-				Virtual method which can be overridden to customize the return value of [method set].
-				Sets a property. Returns [code]true[/code] if the [param property] exists.
+				Override this method to customize the behavior of [method set]. Should set the [param property] to [param value], and return [code]true[/code] if the [param property] exists.
 			</description>
 		</method>
 		<method name="_to_string" qualifiers="virtual">
 			<return type="String" />
 			<description>
-				Virtual method which can be overridden to customize the return value of [method to_string], and thus the object's representation where it is converted to a string, e.g. with [code]print(obj)[/code].
-				Returns a [String] representing the object. If not overridden, defaults to [code]"[ClassName:RID]"[/code].
+				Override this method to customize the return value of [method to_string], and therefore the object's representation as a [String].
+				[codeblock]
+				func _to_string():
+				    return "Welcome to Godot 4!"
+
+				func _init():
+				    print(self)       # Prints Welcome to Godot 4!"
+				    var a = str(self) # a is "Welcome to Godot 4!"
+				[/codeblock]
 			</description>
 		</method>
 		<method name="add_user_signal">
 			<return type="void" />
 			<param index="0" name="signal" type="String" />
 			<param index="1" name="arguments" type="Array" default="[]" />
 			<description>
-				Adds a user-defined [param signal]. Arguments are optional, but can be added as an [Array] of dictionaries, each containing [code]name: String[/code] and [code]type: int[/code] (see [enum Variant.Type]) entries.
+				Adds a user-defined [param signal]. Optional arguments for the signal can be added as an [Array] of dictionaries, each defining a [code]name[/code] [String] and a [code]type[/code] [int] (see [enum Variant.Type]). See also [method has_user_signal].
+				[codeblock]
+				add_user_signal("hurt", [
+				    { "name": "damage", "type": TYPE_INT },
+				    { "name": "source", "type": TYPE_OBJECT }
+				])
+				[/codeblock]
 			</description>
 		</method>
 		<method name="call" qualifiers="vararg">
 			<return type="Variant" />
 			<param index="0" name="method" type="StringName" />
 			<description>
-				Calls the [param method] on the object and returns the result. This method supports a variable number of arguments, so parameters are passed as a comma separated list. Example:
+				Calls the [param method] on the object and returns the result. This method supports a variable number of arguments, so parameters can be passed as a comma separated list.
```

Comment:
Any chance we could get an example here? The examples added elsewhere were really helpful, and this one feels subtle enough to warrant one :)